prow/config/jobs.go

```diff
@@ -198,10 +207,8 @@ func (c *Config) AllJobNames() []string {
 	for _, v := range c.Postsubmits {
 		res = append(res, listPost(v)...)
 	}
-	for _, v := range c.Periodics {
-		for _, j := range v {
-			res = append(res, j.Name)
-		}
+	for _, j := range c.Periodics {
```

Comment:
This method makes it possible to test that all testgrid tabs have a job in Jenkins or prow.